catalog/ui/search-ui/standard/src/main/webapp/js/view/preferences/PreferencesModal.view.js

```diff
@@ -14,37 +14,227 @@
  **/
 /* global define, require */
 define([
+        'application',
         'icanhaz',
         'underscore',
         'marionette',
         'backbone',
         'jquery',
+        'cesium',
+        'maptype',
         'js/view/Modal',
         'text!templates/preferences/preferences.modal.handlebars',
-        'modelbinder'
```

Comment:
You should leave the modelbinder in here. Otherwise, if modelbinder hasn't already been required in by some other module, this one will break down below when model binder is actually used.